### PR TITLE
DOCS: clarify that GTG is lossless

### PR DESCRIPTION
**Pull request by ValZapod**
That is of course depends on #3032 that you so radically closed.

### CODE REVIEW

File: docs/source/specifications/geodetictiffgrids.rst [hunk 1]
```diff
@@ -636,7 +636,7 @@ It contains 5 subgrids. All essential metadata to list the subgrids and their
 georeferencing is contained within the first 3 KB of the file.
 
 The file size is 4.8 MB using DEFLATE compression and floating-point predictor.
-To be compared with the 83 MB of the original .gsb file.
+To be compared with the 83 MB of the original .gsb file that is lossless with it.
```
Comment (rouault) [inline docs/source/specifications/geodetictiffgrids.rst]: I'm not sure this formulation is very clear. I'd rather change the previous line to "The file size is 4.8 MB using DEFLATE compression and floating-point predictor, which are lossless compression parameters"

Comment (ValZapod) [reply docs/source/specifications/geodetictiffgrids.rst:639]: > which are lossless compression parameters"

No, what i meant is that it is losssless with original gsb file, not that the float compressed TIFF is lossless. That is obvious.

Comment (rouault) [inline docs/source/specifications/geodetictiffgrids.rst]:
```suggestion
It results from a lossless conversion of the original .gsb file whose size is 83 MB.
```

**Comment by ValZapod** [comment]
Sigh, will squash it. So, you are positive conversion is lossless?

**Comment by ValZapod** [comment]
Can you recreate bitperfect original gsb file from it?

**Comment by rouault** [comment]
> So, you are positive conversion is lossless?

yes, at least as done by the https://github.com/OSGeo/PROJ-data/blob/master/grid_tools/ntv2_to_gtiff.py script

**Comment by hobu** [comment]
@ValZapod while I'm sure we are all obviously stupider than you, please try to improve your tone and interaction with us. GitHub tickets are not a support venue, and without some example code showing a specific issue on this particular topic, you are not likely to get the satisfaction you are trying to obtain through this venue.

You might try the mailing list for a wider audience, but please remember we are neither your emergency support helpline nor your bug chasers. I understand language and cultural differences can make communication challenging, but please stop acting as if your time is more valuable than ours and we have somehow wasted it with a bug you can neither illustrate or describe.

**Comment by ValZapod** [comment]
> some example code showing a specific issue on this particular topic,

There is no issue. This is just docs change.

> are all obviously stupider than you

That is or opinion only.

**Comment by ValZapod** [comment]
Thanks! I almost git reset'ed it to squash :) Anyway, thanks for your emergency support helpline. I was just flustrated that 16 bit quantised tiff is not lossless and thus gives errors in https://lists.osgeo.org/pipermail/proj/2019-December/009121.html (yep, I knew there was a mailing list).

**Pull request merged**